src/Microsoft.PowerShell.Commands.Utility/commands/utility/GetHash.cs

```diff
@@ -156,6 +156,15 @@ protected override void ProcessRecord()
                         path);
```

Comment:
@iSazonov, your last commit had 1 failures in `PowerShell-CI-windows`
Get-FileHash.Default result tests.Should write non-terminating error if a file is locked
```powershell
Expected strings to be the same, but they were different.
Expected length: 62
Actual length:   61
Strings differ at index 4.
Expected: 'FileReadEr...'
But was:  'FileNotFou...'
at <ScriptBlock>, D:\a\1\s\test\powershell\Modules\Microsoft.PowerShell.Utility\Get-FileHash.Tests.ps1: line 35
35:             $errorVariable.FullyQualifiedErrorId | Should -BeExactly "FileReadError,Microsoft.PowerShell.Commands.GetFileHashCommand"
```